Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.yml

```
@@ -7,29 +10,36 @@ configuration:
   display: Fetch incidents
   required: false
   type: 8
+  section: Collect
 - name: address
   display: |-
     Prisma Cloud Compute Console URL and Port
     (e.g., https://example.net:1234)
   required: true
   type: 0
   additionalinfo:
+  section: Connect
 - name: project
   display: Prisma Cloud Compute Project Name (if applicable)
   required: false
   defaultvalue:
   type: 0
   additionalinfo:
+  section: Connect
```

```suggestion
  section: Connect
  advanced: true
```
Because it is not always applicable